tests/unit/src/main/java/org/mule/functional/junit4/matchers/ClassNameMatcher.java

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright (c) MuleSoft, Inc.  All rights reserved.  http://www.mulesoft.com
+ * The software in this package is published under the terms of the CPAL v1.0
+ * license, a copy of which has been included with this distribution in the
+ * LICENSE.txt file.
+ */
+package org.mule.functional.junit4.matchers;
+
+import org.hamcrest.Description;
+import org.hamcrest.Factory;
+import org.hamcrest.Matcher;
+import org.hamcrest.TypeSafeMatcher;
+
+/**
+ * Matcher to check the class name of an {@link Object}
+ * @param <T> the type of {@link Object} to match
+ */
+public class ClassNameMatcher<T> extends TypeSafeMatcher<T> {
```

Comment:
where is this used? Also, the format is wrong

Reply:
Sorry for the unformatted code. It is used by our integration tests.